### ISSUE TITLE
Curve Frame qL SOP: add summed segment length?

### ISSUE THREAD

**Issue by johnnyquest** [qLab/qLib#1005]

**Comment by gadfly16** [comment]
Do you mean cumulative? I just needed that the other day. Would be great.

**Comment by johnnyquest** [comment]
@gadfly16 Yep, cumulative. I'll add this. I'm also thinking of adding twist-along-curve options. H18 has an Orientation from Curve SOP though. (Still I feel like it wouldn't hurt retaining our own version for a while)
